setup.py

```diff
@@ -55,25 +54,12 @@
         ],
         "testing": [
             "coverage",
-            "pytest>=3.6,<4",
+            "pytest~=6.0.1",
             "pytest-cov",
-            "beautifulsoup4",
-            "pytest-regressions",
+            "pytest-regressions~=2.0.1",
         ],
+        "live-dev": ["sphinx-autobuild", "web-compile~=0.2.1"],
     },
     entry_points={"sphinx.html_themes": ["sphinx_book_theme = sphinx_book_theme"]},
-    package_data={
```

Comment:
this is unnecessary because we have the MANIFEST file right?

Reply:
yep 👍